recipes/ocp/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "ocp" %}
+{% set version = "7.5.3" %}
+{% set occt_ver = "7.5.3" %}
+{% set ocp_tweak = "0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  # currently only used for the license file
+  - url: https://github.com/CadQuery/OCP/archive/refs/tags/{{ occt_ver }}.{{ ocp_tweak }}.tar.gz
+    folder: ocp
+    md5: 15c6c85c616462d5b70356549a6446f1
+
+  # binding sources
+  - url: "https://github.com/CadQuery/OCP/releases/download/{{ occt_ver }}.{{ ocp_tweak }}/OCP_src_windows-2019.zip"  # [win]
+    md5: ffc3f7da4f9369abc53dc6c744f03a1c  # [win]
+    folder : src  # [win]
+  - url: "https://github.com/CadQuery/OCP/releases/download/{{ occt_ver }}.{{ ocp_tweak }}/OCP_src_macOS-10.15.zip"   # [osx]
+    md5: 603735efa2341256a9e5c17e45cecf03  # [osx]
+    folder : src  # [osx]
+  - url: "https://github.com/CadQuery/OCP/releases/download/{{ occt_ver }}.{{ ocp_tweak }}/OCP_src_Ubuntu-18.04.zip"  # [linux]
+    md5: 4cd6a03a5c4f52fff7369829d62c53fe  # [linux]
+    folder : src  # [linux]
+
+build:
+  number: 0
+  skip: true  # [py != 39]
```

Comment:
maybe a comment here...  thought you had mentioned it _could_ work with other pythons, but was just timing out...